R/mainTests.r

```diff
@@ -1627,3 +1628,34 @@ indicator_dp_validation <- function(meta) {
 
   return(output)
 }
+
+# indicator_dp_completed -------------------------------------
+# indicator_dp should be completed for indicators
+
+indicator_dp_completed <- function(meta) {
+  blankIndicators <- meta %>%
+    filter(col_type == "Indicator") %>%
+    filter(is.na(indicator_dp)) %>%
+    pull(col_name)
+
+  if (length(blankIndicators) == 0) {
+    output <- list(
+      "message" = "The indicator_dp column is completed for all indicators.",
+      "result" = "PASS"
+    )
+  } else {
+    if (length(blankIndicators) == 1) {
+      output <- list(
+        "message" = paste0("The following indicator does not have a specified number of dp in the metadata file: '", paste(blankIndicators, collapse = "', '"), "', this should be explicity stated where possible."),
```

Comment:
Should it refer to indicator_dp specifically? Just wondering if analysts might miss where they're meant to add it.

Reply:
How's this? It's getting to be a lot of text 

![image](https://user-images.githubusercontent.com/52536248/100249768-d7c8d480-2f34-11eb-85b6-080164ec382e.png)

Reply:
I think that's more informative yeah 👍